planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -87,19 +87,30 @@ std::pair<bool, bool> NormalLaneChange::getSafePath(LaneChangePath & safe_path)
   return {true, found_safe_path};
 }
 
+LaneChangePath NormalLaneChange::getLaneChangePath() const
+{
+  return isAbortState() ? *abort_path_ : status_.lane_change_path;
+}
+
 BehaviorModuleOutput NormalLaneChange::generateOutput()
 {
   BehaviorModuleOutput output;
   output.path = std::make_shared<PathWithLaneId>(getLaneChangePath().path);
-
   extendOutputDrivableArea(output);
+  output.reference_path = std::make_shared<PathWithLaneId>(getReferencePath());
+  output.turn_signal_info = updateOutputTurnSignal();
 
   if (isAbortState()) {
     return output;
   }
 
   if (isStopState()) {
-    const auto stop_point = utils::insertStopPoint(0.1, *output.path);
+    const auto current_velocity = planner_data_->self_odometry->twist.twist.linear.x;
```

Comment:
```suggestion
    const auto current_velocity = getEgoVelocity();
```
Suggestion on using available getter function